beetsplug/importhistory.py

```diff
@@ -54,7 +54,7 @@ def import_stage(self, _, task):
                 )
                 continue
             item["source_path"] = item.path
-            item.try_sync(write=True, move=False)
+            item.try_sync(write=False, move=False)
```

Comment:
This is surprising! My original purpose here was to write to the database the `source_path` and not to the imported item - I didn't interpret correctly the item terminology.

Reply:
Thanks again @JOJ0 for the detailed answer :).